p2p_wallet/SwiftUI/KeyboardHelper/KeyboardReadable.swift

```diff
@@ -0,0 +1,22 @@
+import Combine
+import UIKit
+
+/// Publisher to read keyboard changes.
+protocol KeyboardReadable {
+    var keyboardPublisher: AnyPublisher<Bool, Never> { get }
```

Comment:
The name doesn't indicate the type of the property. Should be something like `keyboardWillShowPublisher` or `keyboardVisibilityPublisher`

Reply:
agree